File: 프로그래머스/0/120834. 외계행성의 나이/외계행성의 나이.java
class Solution {
    public String solution(int age) {
        String answer = "";  
        String s = String.valueOf(age); //숫자->문자열
        String[] arr = s.split("");  
        
        for(int i=0 ; i<arr.length; i++){
            answer += ( (char)(Integer.parseInt(arr[i])+97));   // 97 == 'a'
        }
        
        return answer;
    }
}

File: 프로그래머스/0/120834. 외계행성의 나이/외계행성의 나이.cpp
#include <string>
#include <vector>

using namespace std;

string solution(int age) {   // age <= 1000 
    string answer =  to_string(age); 
 
    for(int i=0; i<answer.size(); i++){
        answer[i] += char(49);
    }

    return answer;
}